uint16 SucSen0
uint16 SucSen1
